README.md

```diff
@@ -41,11 +41,21 @@ This temporary filessytem will be shared with the target service via the `JoinsN
         # cannot be reloaded.
         changeAction = "restart";
 
+        # The octal mode of /run/keys/environment/EnvFile.
+        # Defaults to 0400.
+        # NOTE: The owner and group of the file are set based on the
+        # infected service's User= and Group= systemd directives.
+        perms = "0400";
```

Comment:
I'm not sure this is necessary, since the file is only ever read by systemd as root. We probably shouldn't document this option, since it is an implementation detail.